brainscore/submission/evaluation.py

```diff
@@ -102,48 +121,69 @@ def run_evaluation(config_file, work_dir, jenkins_id, db_secret, models=None,
                         'raw_result': 0, 'ceiled_result': 0,
                         'error': error, 'finished_time': datetime.datetime.now()
                     })
+                    if scoreInst:
+                        scoreInst.comment = error
```

Comment:
following the change with layers, this might then have to be `comment = error if comment is None else f"{comment}\n{error}` to append to the comment?

Reply:
NO because, this comment is stored in the database, the other comment is only sent to the submitter in the csv file. We are not storing the layer commitment in the database

Reply:
aah gotcha, I thought the other comment was also for the database.
Models now have a comment field though, so could we store the layer commitment there?